types/chainLock.go

```diff
@@ -0,0 +1,123 @@
+package types
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"github.com/tendermint/tendermint/crypto"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+type ChainLock struct {
+	CoreBlockHeight    uint32          `json:"core_block_height,string,omitempty"`   // height of Chain Lock.
+	CoreBlockHash      []byte         `json:"core_block_hash,string,omitempty"`     // hash of Chain Lock.
```

Comment:
`BlockHash`?

Reply:
There are 2 block hashes, one core and one platform, expressing which one I think makes this more easy to understand.